EVE.pm

```diff
@@ -0,0 +1,215 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2022] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+ Ensembl <http://www.ensembl.org/info/about/contact/index.html>
+
+=cut
+
+=head1 NAME
+
+ EVE
+
+=head1 SYNOPSIS
+
+ cp EVE.pm ${HOME}/.vep/Plugins
+ ./vep -i variations.vcf --plugin EVE,file=/path/to/eve/data.vcf.gz
+
+=head1 DESCRIPTION
+
+ This is a plugin for the Ensembl Variant Effect Predictor (VEP) that
+ adds information from EVE (evolutionary model of variant effect).
+ It is only available for GRCh38.
+
+ Please cite EVE publication alongside the VEP if you use this resource:
+ https://www.nature.com/articles/s41586-021-04043-8
+
+###################################################
+# Bash script to merge all VCFs from EVE dataset. #
+###################################################
+
+### BEGIN
+
+# Data link: https://evemodel.org/api/proteins/bulk/download/
```

Comment:
Suggesting that you change this to 
"EVE input file can be downloaded from  https://evemodel.org/api/proteins/bulk/download/ "
For uniformity with other plugins

Reply:
Add this info.